cosmo_tester/test_suites/bootstrap_based_tests/cli_test.py

```diff
@@ -129,6 +142,22 @@ def test_cli_on_rhel_6(cli_package_tester, attributes):
     cli_package_tester.run_test()
 
 
+def test_cli_on_osx(osx_cli_package_tester, attributes):
+    osx_cli_package_tester.inputs.update({
+        'manager_image': attributes.centos_7_AMI,
+        'manager_flavor': attributes.large_AWS_type,
+        'manager_user': attributes.centos_7_username,
+        'aws_access_key': os.environ["AWS_ACCESS_KEY_ID"],
+        'aws_secret_key': os.environ["AWS_ACCESS_KEY"],
+        'mac_public_ip': os.environ["MACINCLOUD_HOST"],
```

Comment:
Can we perhaps change these to 'osx'/'OSX' rather than 'mac'/'MAC', to standardise with what it's referred to as in the rest of the PR?

Reply:
Sure. It's was just closer to the service it's hosted on "macincloud"
But OSX makes more sense